### ISSUE TITLE
[security] Go 1.19.3 and Go 1.18.8 pre-announcement

### ISSUE THREAD

**Issue by GoliathLabs** [robbertkl/docker-ipv6nat#80]
@robbertkl we might need to update the used docker image as well depending on the severity of the vuln. in the standard library.

https://groups.google.com/g/golang-announce/c/dRtDK7WS78g

**Comment by doc75** [comment]
I tried to update all dependencies and base image (cf. https://github.com/doc75/docker-ipv6nat/tree/image_dep_update) but the image is failing right after start with an error message where iptables was complaining about the DOCKER chain not existing.

Running current container with old dependencies is working well on the same environment.

Anyone tried to update the image ?

**Comment by doc75** [comment]
@GoliathLabs does it mean that this repo is unmaintained ?

Having such old image version is not really safe in terms of security.

I am not trying to blame anyone as this this open source work on spare time I guess. I am just trying to have a view on the situation.

**Comment by robbertkl** [comment]
Correct, I'm not actively maintaining this. However, I'm open to an update PR and happy to push out a new release for it.
